src/utils/did.js

```diff
@@ -3,15 +3,128 @@
 // Import some utils from Polkadot JS
 // eslint-disable-next-line max-classes-per-file
 import { randomAsHex, encodeAddress } from '@polkadot/util-crypto';
+import { u8aToHex } from '@polkadot/util';
 import { isHexWithGivenByteSize, getHexIdentifier } from './codec';
+import { PublicKeyEd25519, PublicKeySecp256k1, PublicKey, VerificationRelationship } from '../public-keys';
 
-import { Signature } from '../signatures'; // eslint-disable-line
-import { PublicKey, VerificationRelationship } from '../public-keys'; // eslint-disable-line
+import { Signature } from "../signatures"; // eslint-disable-line
+import {
+  getPublicKeyFromKeyringPair,
+  getSignatureFromKeyringPair,
+  getStateChange,
+} from './misc';
 
 export const DockDIDMethod = 'dock';
 export const DockDIDQualifier = `did:${DockDIDMethod}:`;
+export const DockDIDMethodKeyQualifier = `did:key:${DockDIDMethod}:`;
```

Comment:
The full DID would be `did:key:dock:<value>` but as per spec https://w3c-ccg.github.io/did-method-key/, full DID should be `did:key:<value>`